backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -65,12 +65,14 @@ public class OrgMutations
         Guid projectId)
     {
         var org = await dbContext.Orgs.FindAsync(orgId);
-        NotFoundException.ThrowIfNull(org);
+        // old: NotFoundException.ThrowIfNull(org);
+        if (org is null) throw new NotFoundException("Organization not found", "org");
```

Comment:
This change does the exact same thing as the code it replaces, except that instead of "org" it will have "Organization". Check the NotFoundException.cs file and see if you can understand exactly what it does.

This change is not necessary, and we would prefer the old version.